lib/asyncapiSchemaFormatParser.js

```diff
@@ -80,12 +84,18 @@ function getValidator(version) {
  * 
  * @private
  * @param {Object} asyncapiSchema AsyncAPI specification JSON Schema
+ * @param {Object} version AsyncAPI version.
  * @returns {Object} valid JSON Schema document describing format of AsyncAPI-valid schema for message payload
  */
-function preparePayloadSchema(asyncapiSchema) {
+function preparePayloadSchema(asyncapiSchema, version) {
+  const payloadSchema = `http://asyncapi.com/definitions/${version}/schema.json`;
```

Comment:
I'd use the HTTPS URL, otherwise, it will have to resolve the redirection first, making it slower.

```suggestion
  const payloadSchema = `https://asyncapi.com/definitions/${version}/schema.json`;
```

Reply:
Ok base my pr on: https://github.com/asyncapi/parser-js/pull/423
was not the best idea. But i am unable to wait to get it merged.
This change was introduced with #423